src/TermInterface.jl

```diff
@@ -1,18 +1,32 @@
 module TermInterface
 
 """
-  istree(x)
+    iscall(x)
+Returns `true` if `x` is a function call expression. If true, `operation`, `arguments`
+must also be defined for `x`.
+"""
+iscall(x) = false
+export iscall
+
+Base.@deprecate_binding istree iscall
+"""
+   istree(x)
 
-Returns `true` if `x` is a term. If true, `operation`, `arguments`
-must also be defined for `x` appropriately.
+Alias of `iscall`
 """
-istree(x) = false
-export istree
+istree
```

Comment:
Can't we just remove and do a major version bump?

Reply:
Maybe.

Reply:
We definitely need a major version bump

Reply:
Removed